code/style.css

```diff
@@ -6,145 +6,212 @@ body {
-  padding: 20px 24px;
-  padding-top: 0;
+  padding: 20px;
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
+  justify-content: space-between;
+}
+
+.form {
+  width: 100%;
+  display: flex;
+  justify-content: center;
 }
 
+/* everything belonging to the chat*/
+
 .chat {
   flex: 1;
   display: flex;
   justify-content: flex-start;
-  overflow: scroll;
+  overflow-y: scroll;
   flex-direction: column;
   padding-bottom: 16px;
 }
 
-.bot-msg {
-  display: flex;
-  margin: 16px 8px 0 0;
-  flex-shrink: 0;
+.bubble {
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 26px;
+  padding: 4px 4px;
+  color: #0c1546;
+  max-width: 50%;
 }
 
-.user-msg {
+.user-message {
   display: flex;
   justify-content: flex-end;
   margin: 16px 0 0 8px;
   flex-shrink: 0;
 }
 
-.bot-msg img,
-.user-msg img {
-  width: 60px;
-  height: 60px;
+.user-message-img,
+.bot-message-img {
+  width: 50px;
+  height: 50px;
+  margin: 3px;
 }
 
-.bubble {
-  background: #e5e9ff;
-  font-weight: 600;
-  font-size: 16px;
-  line-height: 26px;
-  padding: 16px 24px;
-  color: #0026ff;
-  max-width: 40%;
+.user-bubble {
+  border-radius: 26px 0 26px 26px;
+  margin-right: 8px;
+  background: #79aab5;
+}
+
+.bot-message {
+  display: flex;
+  margin: 16px 8px 0 0;
+  flex-shrink: 0;
+
 }
 
 .bot-bubble {
   border-radius: 0px 26px 26px 26px;
   margin-left: 8px;
+  background: yellow;
 }
 
-.user-bubble {
-  border-radius: 26px 0 26px 26px;
-  margin-right: 8px;
-}
-
-.input-wrapper {
+/* In the following are the different forms listed:*/
+.form {
   display: flex;
   justify-content: center;
+  align-items: center;
+  width: 100%;
 }
 
-.input-wrapper form {
-  width: 100%;
-  display: flex;
-  align-items: center;
+/*The first button ("Yes!") to continue the conversation */
+
+.confirm-button {
+  background-color: rgb(4, 38, 52);
+  color: white;
+  font-weight: bold;
+  font-size: 30px;
+  width: 80%;
+  min-height: 50px;
+  border-radius: 10%;
+  cursor: pointer;
+  border: solid, white;
+  overflow: hidden;
 }
 
-label {
-  font-size: 16px;
-  font-family: 'Montserrat';
-  font-weight: 500;
-  color: #0026ff;
-  margin-right: 20px;
+.confirm-button:hover {
+  background-color: #79aab5;
 }
 
-button {
-  background-color: #0026ff;
+/* Button to send in name */
+.input-button {
+  background-color: rgb(4, 38, 52);
   color: white;
+  font-weight: bold;
+  font-size: 20px;
+  width: 60px;
+  height: 40px;
+  border-radius: 10px;
+  cursor: pointer;
+  border: solid, white;
+  overflow: hidden;
+  margin: 10px;
+}
+
+/* The input field to write the name */
+.input {
+  box-sizing: border-box;
   border: none;
   border-radius: 4px;
-  padding: 16px 20px;
-  margin-right: 4px;
+  background: white;
+  color: rgb(4, 38, 52);
+  padding: 18px;
+  font-family: 'Moneserrat', sans-serif;
+  font-weight: 600;
+  font-size: 18 px;
+  line-height: 10px;
+  margin: none;
+  flex: 1;
+}
+
+/* buttons to choose the food category */
+.foodtype-container,
```

Comment:
Since these containers and buttons seem to have the exact same CSS, perhaps you could have just used the same class on each element instead of having unique class names for each.